db/migrations/20221129064741_writing_images.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE writing_images (
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+writing_id INT NOT NULL,
+image_url  VARCHAR(2500)  NOT NULL,
+CONSTRAINT author_id FOREIGN KEY (writing_id) REFERENCES writings(id)
```

Comment:
```suggestion
CONSTRAINT ... FOREIGN KEY (writing_id) REFERENCES writings(id)
```
`writing_id`를 참조하고 있는데 네이밍은 `author_id`네요. 너무 당연한 것들에 대해서는 suggestion 남기지 않을테니 고민해보시고 수정해주세요.